DEV_ENVIRONMENT.md

```diff
@@ -6,22 +6,21 @@
    - On Windows: Follow the [instructions](https://docs.bazel.build/versions/master/install-windows.html#5-optional-install-compilers-and-language-runtimes)
    to install [`MSYS2`](https://www.msys2.org/) and the listed "Common MSYS2 packages".
    Afterwards add `C:\msys64\usr\bin` to the `PATH` environment variable.
-3. Run `yarn global add gulp` to install gulp.
-4. Fork the `angular/components` repo on GitHub.
-5. Clone your fork to your machine with `git clone`.
+3. Fork the `angular/components` repo on GitHub.
+4. Clone your fork to your machine with `git clone`.
    Recommendation: name your git remotes `upstream` for `angular/components`
    and `<your-username>` for your fork. Also see the [team git shortcuts](https://github.com/angular/components/wiki/Team-git----bash-shortcuts).
-6. From the root of the project, run `yarn`.
+5. From the root of the project, run `yarn dev-app`.
 
 
-To build angular/components in dev mode, run `gulp material:build`.
-To build angular/components in release mode, run `gulp material:build-release`
+To build angular/components in dev mode, run `yarn gulp material:build`.
+To build angular/components in release mode, run `yarn gulp material:build-release`
```

Comment:
These are no longer up-to-date. They should run `yarn build` which runs the `build-packages-dist.js` script. We should also mention that the release output can be found in `dist/releases`.